sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/EnableCosmosAuditing.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Since all the other annotations are in `com/azure/spring/data/cosmos/core/mapping/` directory, I think we should also place this class in the same directory to be consistent with other Annotations.

Reply:
@Blackbaud-EricSlater Any plans on updating this class package to `core/mapping` ?